scripts/MRK17_Exp1_rk00.jmd

```diff
@@ -40,21 +40,21 @@ are basically default. We do not look at fixed effects during model selection.
 After model selection we look at all estimates.
 
 ```julia
-show(oviLMM)
+oviLMM
 ```
 
 # Appendix 
 
 ## Weave the document in the REPL
 
 + using Weave
-+ weave("notebooks/MRK17_Exp1_rk00.jmd", doctype="md2html")
++ weave(scriptsdir("MRK17_Exp1_rk00.jmd"), doctype="md2html")
```

Comment:
For accessing the files in `scripts/` it is easy to use the `scriptsdir` function.  For files in `notebooks/` I am using the more awkward `projectdir("notebooks","filename.jmd")` but we can replace that with a `notebookdir` function if desired.